ui/imports/shared/popups/send/SendModal.qml

```diff
@@ -219,20 +227,6 @@ StatusDialog {
         if(!!popup.preDefinedAmountToSend) {
             amountToSendInput.input.text = Number(popup.preDefinedAmountToSend).toLocaleString(Qt.locale(), 'f', -128)
         }
-
-        if(!!popup.preSelectedRecipient) {
-            recipientLoader.selectedRecipientType = popup.preSelectedRecipientType
-            if (popup.preSelectedRecipientType == TabAddressSelectorView.Type.Address) {
-                recipientLoader.selectedRecipient = {address: popup.preSelectedRecipient}
-            } else {
-                recipientLoader.selectedRecipient = popup.preSelectedRecipient
-            }
-        }
-
-        if(d.isBridgeTx) {
-            recipientLoader.selectedRecipientType = TabAddressSelectorView.Type.Address
-            recipientLoader.selectedRecipient = {address: popup.preSelectedAccount.address}
-        }
```

Comment:
Why are these 2 codeblocks removed from here?

Reply:
Now we have an alias between `preSelectedRecipient` and `preSelectedRecipientType` and the `recipientLoader` properties. There's no need of manual assignements.

- The bridge use case is covered in: `WalletLayout.qml` line _334_.
- The send modal use cases were already covered by all the _open send entry points_. Maybe some minor adjustment done here to fullfill the complete flow.